### PR TITLE
chore: launch week 3 - community day

### PR DESCRIPTION
**Pull request by thorwebdev**
moved to https://github.com/supabase/supabase/pull/4135

### CODE REVIEW

File: www/_blog/2021-11-29-community-day.mdx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+And below are some of the community one's we've come across recently. If you've created one yourself, please notify us by tagging us in a Tweet!
+
+- [Supabase Auth With Next.Js 12 Middleware](https://jitsu.com/blog/supabase-nextjs-middleware#what-were-going-to-build)
+- [Vue 3 & Supabase | Workout Tracker App](https://www.youtube.com/watch?v=3tF0fGkd4ho)
+- [Use Supabase Auth with Vue.js 3](https://vueschool.io/articles/vuejs-tutorials/use-supabase-auth-with-vue-js-3/)
+- [Dynamic Jamstack with Stencil and Supabase](https://ionicframework.com/blog/dynamic-jamstack-with-stencil-and-supabase)
+
+## User stories
+
+Anytime we scroll through [https://www.madewithsupabase.com/](https://www.madewithsupabase.com/) we're absolutely blown away by the awesome things y'all are building with Supabase.
```
Comment (kiwicopple) [inline www/_blog/2021-11-29-community-day.mdx:104]:
```suggestion
Anytime we scroll through [madewithsupabase.com](https://www.madewithsupabase.com/) we're absolutely blown away by the awesome things y'all are building with Supabase. 
```

File: www/_blog/2021-11-29-community-day.mdx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+- New maintainers [`@Dreinon`](https://github.com/dreinon), [`@anand2312`](https://github.com/anand2312) and [`@leynier`](https://github.com/leynier). 💚
+- Gotrue-py rewritten and now at feature parity with gotrue-js.
+- Postgrest-py now support synchronous operations as well instead of just synchronous operations in the past. Also at feature-parity with postgrest-js.
+- Supabase-py is now Supabase (e.g. you do `pip3 install supabase` instead of `pip3 install supabase-py`).
+- Storage is working for the py client library but it's yet to be extracted out as a standalone lib.
+
+## New tutorials and integration guides
+
+Our own [Jon Meyers](https://twitter.com/_dijonmusters) and the broader community have been busy creating awesome tutorials and integration guides to help everyone build even more amazing things with Supabase. Jon has added official guides for working with [Auth0](https://supabase.com/docs/guides/integrations/auth0), [Draftbit](https://supabase.com/docs/guides/integrations/draftbit), [Prisma](https://supabase.com/docs/guides/integrations/prisma), and [Vercel](https://supabase.com/docs/guides/integrations/vercel)!
```
Comment (dijonmusters) [inline www/_blog/2021-11-29-community-day.mdx:93]: Just a small one, I think the Prisma guide was @MildTomato, and the Draftbit one was from @amandeepmittal. Not sure if he is from draftbit or just the Supabase community.

Maybe change to:

Our own [Jon Meyers](https://twitter.com/_dijonmusters) and the broader community have been busy creating awesome tutorials and integration guides to help everyone build even more amazing things with Supabase. Jon has added official guides for working with [Auth0](https://supabase.com/docs/guides/integrations/auth0) and [Vercel](https://supabase.com/docs/guides/integrations/vercel), [Jonny Summers-Muir](https://twitter.com/JSummersMuir) has been busy paving the way for [Prisma](https://supabase.com/docs/guides/integrations/prisma), and [Aman Mittal](https://github.com/amandeepmittal) has hooked us up with a guide for [Draftbit](https://supabase.com/docs/guides/integrations/draftbit)!

File: www/_blog/2021-11-29-community-day.mdx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+authorURL: https://github.com/thorwebdev
+image: launch-week-three/community-day/supabase-oss.jpg
+thumb: launch-week-three/community-day/supabase-oss.jpg
+tags:
+  - launch-week
+date: '2021-11-29'
+toc_depth: 2
+---
+
+## Blog
```
Comment (kiwicopple) [inline www/_blog/2021-11-29-community-day.mdx:16]:
```suggestion
```

File: www/_blog/2021-11-28-postgrest-9.mdx [hunk 4]
```diff
@@ -0,0 +1,106 @@
+---
+title: 'PostgREST v0'
+description: Kicking off launch week by highlighting the communities around Supabase.
+author: steve_chavez
+author_url: https://github.com/steve-chavez
+author_image_url: https://github.com/steve-chavez.png
+authorURL: https://github.com/steve-chavez
+image: launch-week-three/community-day/supabase-oss.jpg
+thumb: launch-week-three/community-day/supabase-oss.jpg
```
Comment (kiwicopple) [inline www/_blog/2021-11-28-postgrest-9.mdx:9]: @MildTomato - could we please do a PostgREST 9 OG like you did for pg 14?

Comment (MildTomato) [reply www/_blog/2021-11-28-postgrest-9.mdx:9]: adding now

Comment (kiwicopple) [reply www/_blog/2021-11-28-postgrest-9.mdx:9]: I saw this was resolved - but it looks like it's still pending?

File: www/_blog/2021-11-28-postgrest-9.mdx [hunk 5]
```diff
@@ -0,0 +1,106 @@
+---
+title: 'PostgREST 9'
+description: New features and updates in PostgREST version 9.
+author: steve_chavez
+author_url: https://github.com/steve-chavez
+author_image_url: https://github.com/steve-chavez.png
+authorURL: https://github.com/steve-chavez
+image: launch-week-three/community-day/supabase-oss.jpg
```
Comment (kiwicopple) [inline www/_blog/2021-11-28-postgrest-9.mdx:8]: @thorwebdev just re-tagging this one as I don't think it's resolved?

Comment (thorwebdev) [reply www/_blog/2021-11-28-postgrest-9.mdx:8]: @kiwicopple sorry, got it locally. just ticking off so I know what's left to do. Will push in a sec.

Comment (thorwebdev) [reply www/_blog/2021-11-28-postgrest-9.mdx:8]: I know, not the best workflow 🙈

Comment (kiwicopple) [reply www/_blog/2021-11-28-postgrest-9.mdx:8]: all good!

**Pull request closed without merging**